src/ObjCRuntime/RuntimeOptions.cs

```diff
@@ -19,11 +19,7 @@ namespace Xamarin.Bundler {
 namespace ObjCRuntime {
 #endif
 	class RuntimeOptions {
-#if !LEGACY_TOOLS
```

Comment:
You'll have to remove this file from tools/mtouch/mtouch.csproj, tools/mmp/mmp.csproj and tools/dotnet-linker/dotnet-linker.csproj, and fix/remove any related code to make those projects compile again.